packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -157,8 +181,12 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
         onInput={(e) => this.combobox.onInputChange(e)}
         onKeyUp={(e) => this.combobox.onInputKeyup(e)}
         onKeyDown={(e) => this.combobox.onInputKeydown(e)}
+        tabindex="0"
         type="text"
-        placeholder=""
+        aria-autocomplete="list"
```

Comment:
Shouldn't it be type = "search" on line 184 ?

Reply:
Also probably missing autocomplete=off to prevent browser trying to also show their own autocompletion widget.

Reply:
In all the accessibility examples I saw it was typed to text.